src/data/vapor-pressure-constants.ts

```diff
@@ -1203,6 +1203,291 @@ export const DIPPRVaporPressureDictionary: Record<
+    C2: -6541.6,
+    C3: -12.247,
+    C4: 0.000012311,
+    C5: 2,
+    minimumTemperature: 178.01,
+    maximumTemperature: 510,
+    pressureAtMinimumTemperature: 5.93,
+    pressureAtMaximumTemperature: 6.093e6,
+  },
+
+  '1,1Dichloropropane': {
+    name: '1,1-Dichloropropane',
+    formula: 'C3H6Cl2',
+    CAS: '78-99-9',
+    C1: 83.495,
+    C2: -6661.4,
+    C3: -9.2386,
+    C4: 6.7652e-6,
+    C5: 2,
+    minimumTemperature: 192.5,
+    maximumTemperature: 560,
+    pressureAtMinimumTemperature: 1.72,
+    pressureAtMaximumTemperature: 4.239e6,
+  },
+
+  DiethanolAmine: {
+    name: 'Diethanol amine',
+    formula: 'C4H11NO2',
+    CAS: '111-42-2',
+    C1: 106.38,
+    C2: -13714,
+    C3: -11.06,
+    C4: 3.26e-18,
+    C5: 6,
+    minimumTemperature: 301.15,
+    maximumTemperature: 736.6,
+    pressureAtMinimumTemperature: 1.02e-1,
+    pressureAtMaximumTemperature: 4.26e6,
+  },
+
+  DiethylAmine: {
+    name: 'Diethyl amine',
+    formula: 'C4H11N',
+    CAS: '109-89-7',
+    C1: 49.314,
+    C2: -4949,
+    C3: -3.9256,
+    C4: 9.2e-18,
+    C5: 6,
+    minimumTemperature: 223.35,
+    maximumTemperature: 496.6,
+    pressureAtMinimumTemperature: 3.74e2,
+    pressureAtMaximumTemperature: 3.674e6,
+  },
+
+  DiethylEther: {
+    name: 'Diethyl ether',
+    formula: 'C4H10O',
+    CAS: '60-29-7',
+    C1: 136.9,
+    C2: -6954.3,
+    C3: -19.254,
+    C4: 0.024508,
+    C5: 1,
+    minimumTemperature: 156.85,
+    maximumTemperature: 466.7,
+    pressureAtMinimumTemperature: 3.95e-1,
+    pressureAtMaximumTemperature: 3.641e6,
+  },
+
+  DiethylSulphide: {
+    name: 'Diethyl sulphide',
+    formula: 'C4H10S',
+    CAS: '352-93-2',
+    C1: 46.705,
+    C2: -5177.4,
+    C3: -3.5985,
+    C4: 107147e-6,
+    C5: 2,
+    minimumTemperature: 169.2,
+    maximumTemperature: 557.15,
+    pressureAtMinimumTemperature: 9.93e-2,
+    pressureAtMaximumTemperature: 3.961e6,
+  },
+
+  '1,1Difluoroethane': {
+    name: '1,1-Difluoroethane',
+    formula: 'C2H4F2',
+    CAS: '75-37-6',
+    C1: 73.491,
+    C2: -4385.9,
+    C3: -8.1851,
+    C4: 0.000012978,
+    C5: 2,
+    minimumTemperature: 154.56,
+    maximumTemperature: 386.44,
+    pressureAtMinimumTemperature: 6.45e1,
+    pressureAtMaximumTemperature: 4.507e6,
+  },
+
+  '1,2Difluoroethane': {
+    name: '1,2-Difluoroethane',
+    formula: 'C2H4F2',
+    CAS: '624-72-6',
+    C1: 84.625,
+    C2: -5217.4,
+    C3: -9.871,
+    C4: 0.00001305,
+    C5: 2,
+    minimumTemperature: 179.6,
+    maximumTemperature: 445,
+    pressureAtMinimumTemperature: 1.17e2,
+    pressureAtMaximumTemperature: 4.372e6,
+  },
+
+  Difluoromethane: {
+    name: 'Difluoro amine',
+    formula: 'CH2F2',
+    CAS: '75-10-5',
+    C1: 69.132,
+    C2: -3847.7,
+    C3: -7.5868,
+    C4: 0.000015065,
+    C5: 2,
+    minimumTemperature: 136.95,
+    maximumTemperature: 351.26,
+    pressureAtMinimumTemperature: 5.43e1,
+    pressureAtMaximumTemperature: 5.761e6,
+  },
+
+  DisopropylAmine: {
```

Comment:
```suggestion
  DisoPropylAmine: {
```
@maneetgoyal @sumitnegi7 should we do this way?

Reply:
The current way is fine i guess it is not separate by space so no need to capitalize it.